config/initializers/mime_types.rb

```diff
@@ -2,3 +2,4 @@
 
 # Add new mime types for use in respond_to blocks:
 # Mime::Type.register "text/richtext", :rtf
+Mime::Type.register 'application/pdf', :pdf
```

Comment:
Looks like this already exists:

```
[1] pry » Mime[:pdf]
=> #<Mime::Type:0x000056141c78e020 @synonyms=[], @symbol=:pdf, @string="application/pdf", @hash=-1686108550264282935>
```

Reply:
I did this after I discovered some strange errors in the console running rails, when I was actually trying to resolve the unbelievably slow sass processing. Googling the error message has resolved it being related to this somehow. And since I've put it there I've never seen it again. I presume it wasn't always set before?

Reply:
The SASS pain was eased by replacing sass_rails with sassc_rails. Had I only known that earlier, it'd saved loads of 👋 💥 ☠️ 💣 . Changing stuff in SASS could literally use 40 seconds or more to recompile, while with sassc the difference to having made changes is hardly noticable.